hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/client/rpc/TestContainerReplicationEndToEnd.java

```diff
@@ -167,9 +166,13 @@ public void testContainerReplication() throws Exception {
             .getPipeline(pipelineID);
     key.close();
 
-    if (cluster.getStorageContainerManager().getContainerManager()
-        .getContainer(new ContainerID(containerID)).getState() !=
-        HddsProtos.LifeCycleState.CLOSING) {
+    HddsProtos.LifeCycleState containerState =
+        cluster.getStorageContainerManager().getContainerManager()
+            .getContainer(new ContainerID(containerID)).getState();
+    LoggerFactory.getLogger(TestContainerReplicationEndToEnd.class).info(
+        "Current Container State is " + containerState);
```

Comment:
Please use placeholder in log message instead of `+` to avoid new code warning (sonar).

Reply:
done. Also changed the intervals you mentioned to maintain the same ratio as their default values. for container-report-intrval : stalenode-interval : deadnode-interval.